pandas/_typing.py

```diff
@@ -83,7 +83,7 @@
 PythonScalar = Union[str, int, float, bool]
 DatetimeLikeScalar = Union["Period", "Timestamp", "Timedelta"]
 PandasScalar = Union["Period", "Timestamp", "Timedelta", "Interval"]
-Scalar = Union[PythonScalar, PandasScalar]
+Scalar = Union[PythonScalar, PandasScalar, np.datetime64, np.timedelta64, datetime]
```

Comment:
does `np.datetime64` inherit from `datetime`? Would probably only need one of them.

Reply:
It's not a subclass:
```python
>>> import datetime as dt
>>> import numpy as np
>>> import pandas as pd
>>> issubclass(np.datetime64, dt.datetime)
False
>>> issubclass(pd.Timestamp, np.datetime64)
False
>>> issubclass(pd.Timestamp, dt.datetime)
True
```